Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -272,6 +274,7 @@ MediaPlayerPrivateGStreamer::~MediaPlayerPrivateGStreamer()
 
     m_player = nullptr;
     m_notifier->invalidate();
+    m_abortableTaskQueue.finishAborting();
```

Comment:
This is not necessary. At this point no code will be enqueueing any tasks to the abortable task queue, and even if they did, we wouldn't want them to be executed.